sdk/storage/Azure.Storage.Files.Shares/src/Models/ClosedHandlesResult.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma warning disable SA1402  // File may only contain a single type
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// The result of a force close handle or force close all handles operation.
+    /// </summary>
+    public class ClosedHandlesResult
```

Comment:
@KrzysztofCwalina's proposed name was `CloseHandlesResult` with just `Close` instead of `Closed`